<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VoxelGridWin</class>
 <widget class="QDialog" name="VoxelGridWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>482</width>
    <height>98</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VoxelGridWin</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Leaf size of X:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="xEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Leaf size of Y:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="yEdit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Leaf size of Z:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="zEdit"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="yesBtn">
     <property name="text">
      <string>Yes</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
